kubernetes/container-azm-ms-agentconfig.yaml

```diff
@@ -48,9 +48,11 @@ data:
           # See documentation at https://aka.ms/ContainerLogv2 for benefits of v2 schema over v1 schema before opting for "v2" schema
           # containerlog_schema_version = "v2"
        #[log_collection_settings.enable_multiline_logs]
-          # fluent-bit based multiline log collection for .NET, Go, Java, and Python stacktraces.
+          # fluent-bit based multiline log collection for .NET, Go, Java, and Python stacktraces. Update stacktrace_languages to specificy which languages to collect stacktraces for. Default Go, Java, and Python.
```

Comment:
will this impact dotnet only user? And also do we want to call out dotnet in case we want to extend this support in future?

Reply:
will update